src/draive/conversation/realtime/default.py

```
@@ -1,210 +1,199 @@
+
+    return lmm_recall
 
-                ctx.log_debug("...output processing finished!")
 
-        ctx.spawn(process_output)
+def _to_lmm_remember(
+    remember: MemoryRemembering[ConversationMessage],
+    /,
+) -> MemoryRemembering[LMMContextElement]:
+    async def lmm_remember(
+        *items: LMMContextElement,
```

Pylint (3.3.7)

[warning] Unused argument 'extra'

(W0613)